landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md

```diff
@@ -0,0 +1,144 @@
+   order_summary = transform(order_data)
+   load(order_summary["total_order_value"])
+
+tutorial_etl_dag = tutorial_taskflow_api_etl()
+```
+
+## Fully specified REST API (AIP-32)
+
+We now have a fully supported, no-longer-experimental API with a comprehensive OpenAPI specification
+
+Read more here:
+
+[REST API Documentation](http://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html).
+
+## Massive Scheduler performance improvements
+
+As part of AIP-15 (Scheduler HA+performance) and other work Kamil did, we significantly improved the performance of  the Airflow Scheduler. It now starts tasks much, MUCH quicker.
+
+Over at Astronomer.io we've [benchmarked the scheduler—it's fast](https://www.astronomer.io/blog/airflow-2-scheduler) (we had to triple check the numbers as we don't quite believe them at first!)
+
+## Scheduler is now HA compatible (AIP-15)
+
+It's now possible and supported to run more than a single scheduler instance. This is super useful for both resiliency (in case a scheduler goes down) and scheduling performance.
+
+To fully use this feature you need Postgres 9.6+ or MySQL 8+ (MySQL 5, and MariaDB won't work with more than one scheduler I'm afraid).
+
+There's no config or other set up required to run more than one scheduler—just start up a scheduler somewhere else (ensuring it has access to the DAG files) and it will cooperate with your existing schedulers through the database.
+
+For more information, read the [Scheduler HA documentation](http://airflow.apache.org/docs/apache-airflow/stable/scheduler.html#running-more-than-one-scheduler).
+
+## Task Groups (AIP-34)
+
+SubDAGs were commonly used for grouping tasks in the UI, but they had many drawbacks in their execution behaviour (primarirly that they only executed a single task in parallel!) To improve this experience, we’ve introduced "Task Groups": a method for organizing tasks which provides the same grouping behaviour as a subdag without any of the execution-time drawbacks.
+
+SubDAGs will still work for now, but we think that any previous use of SubDAGs can now be replaced with task groups. If you find an example where this isn't the case, please let us know by opening an issue on GitHub
+
+For more information, check out the [Task Group documentation](http://airflow.apache.org/docs/apache-airflow/stable/concepts.html#taskgroup).
+
+## Refreshed UI
+
+We've given the Airflow UI [a visual refresh](https://github.com/apache/airflow/pull/11195) and updated some of the styling.
+
+![Airflow 2.0's new UI](airflow-2.0-ui.gif)
+
+Check out [the screenshots in the docs](http://airflow.apache.org/docs/apache-airflow/stable/ui.html) for more.
+
+## Smart Sensors for reduced load from sensors (AIP-17)
+
+If you make heavy use of sensors in your Airflow cluster, you might find  that sensor execution takes up a significant proportion of your cluster even with "reshedule" mode. To improve this,  we've added a new mode called "Smart Sensors".
```

Comment:
```suggestion
If you make heavy use of sensors in your Airflow cluster, you might find that sensor execution takes up a significant proportion of your cluster even with "reschedule" mode. To improve this,  we've added a new mode called "Smart Sensors".
```

Reply:
Can't apply this suggestion for some reason.